vars/copyContainer.groovy

```diff
@@ -69,7 +73,7 @@ def craneCopy() {
         sh("set -x && crane cp ${source_registry}/${source_image} ${destination_registry}/${destination_image}")
     }
 
-    sh("set +x && docker logout && docker logout ${destination_registry}")
+    sh("set +x && docker logout && docker logout public.ecr.aws")
```

Comment:
Does docker logout work with ECR? I was under the assumption you need to use aws cli to login and logout of ECR registry.